### PR TITLE
Make library Python 3 compatible

### PR DESCRIPTION
**Pull request by MauroPfister**
Update library to make it compatible with Python 3

Pull in [`py3-chamberconnectlibrary` ](https://github.com/EspecNorthAmerica/ChamberConnectLibrary/tree/py3-chamberconnectlibrary)branch from upstream as a starting point.

**Changes on top of `py3-chamberconnectlibrary` branch :**
- Fix imports
- Fix passing of `kwargs` to P300 and SCP220 clients
- Fix decoding of messages

### CODE REVIEW

File: chamberconnectlibrary/p300.py [hunk 1]
```diff
@@ -121,7 +119,9 @@ def read_date_time(self):
         returns:
             {"year":int,"month":int,"day":int, "hour":int, "minute":int, "second":int}
         '''
-        date, time = self.ctlr.interact(['DATE?', 'TIME?'])
+        #date, time = self.ctlr.interact(['DATE?', 'TIME?'])
+        date = (self.ctlr.interact('DATE?'))
```
Comment (fkaiser) [inline chamberconnectlibrary/p300.py:123]: I see in vs code empty spaces at the end. Do you also see them?

**Review by fkaiser — COMMENTED** [review]
I just reviewed your commits.

File: chamberconnectlibrary/p300.py (hunk 1 above)
Comment (MauroPfister) [reply chamberconnectlibrary/p300.py:123]: Yeah I see them too. However, this is part of the code that I didn't touch at all and I would suggest leaving it as is to keep the difference to upstream as small as possible.

Comment (fkaiser) [reply chamberconnectlibrary/p300.py:123]: all good then.

**Review by fkaiser — APPROVED** [review]

**Pull request merged**